Kopernicus/Kopernicus/Configuration/PropertiesLoader.cs

```diff
@@ -277,6 +277,19 @@ public EnumParser<RDVisibility> hiddenRnD
                 set { celestialBody.Set("hiddenRnD", value.value); }
             }
 
+            // If the body should be hidden in RnD
+            [ParserTarget("RDRotation")]
+            public NumericParser<bool> RDRotation
```

Comment:
Can you change this to `RnDRotation`?

Reply:
should we change **RD**Visibility as well?

or do you want to have **RD**Visibility  and **RnD**Rotation

Reply:
Seems like a good idea, but I wouldn't change it. Instead we should add a second ParserTarget with the new name to it.